src/common/CMakeLists.txt

```diff
@@ -0,0 +1,324 @@
+  dsp/effects/chowdsp/tape/LossFilter.h
+  dsp/effects/chowdsp/tape/ToneControl.cpp
+  dsp/effects/chowdsp/tape/ToneControl.h
+  dsp/filters/AllpassFilter.h
+  dsp/filters/BiquadFilter.cpp
+  dsp/filters/BiquadFilter.h
+  dsp/filters/BiquadFilterSSE2.cpp
+  dsp/filters/DiodeLadder.cpp
+  dsp/filters/DiodeLadder.h
+  dsp/filters/FilterCoefficientMaker.cpp
+  dsp/filters/FilterCoefficientMaker.h
+  dsp/filters/K35Filter.cpp
+  dsp/filters/K35Filter.h
+  dsp/filters/NonlinearFeedback.cpp
+  dsp/filters/NonlinearFeedback.h
+  dsp/filters/NonlinearStates.cpp
+  dsp/filters/NonlinearStates.h
+  dsp/filters/OBXDFilter.cpp
+  dsp/filters/OBXDFilter.h
+  dsp/filters/ThreelerFilter.cpp
+  dsp/filters/ThreelerFilter.h
+  dsp/filters/VectorizedSVFilter.cpp
+  dsp/filters/VectorizedSVFilter.h
+  dsp/filters/VintageLadders.cpp
+  dsp/filters/VintageLadders.h
+  dsp/modulators/ADSRModulationSource.cpp
+  dsp/modulators/ADSRModulationSource.h
+  dsp/modulators/FormulaModulationHelper.cpp
+  dsp/modulators/FormulaModulationHelper.h
+  dsp/modulators/LFOModulationSource.cpp
+  dsp/modulators/LFOModulationSource.h
+  dsp/modulators/MSEGModulationHelper.cpp
+  dsp/modulators/MSEGModulationHelper.h
+  dsp/oscillators/AliasOscillator.cpp
+  dsp/oscillators/AliasOscillator.h
+  dsp/oscillators/AudioInputOscillator.cpp
+  dsp/oscillators/AudioInputOscillator.h
+  dsp/oscillators/ClassicOscillator.cpp
+  dsp/oscillators/ClassicOscillator.h
+  dsp/oscillators/FM2Oscillator.cpp
+  dsp/oscillators/FM2Oscillator.h
+  dsp/oscillators/FM3Oscillator.cpp
+  dsp/oscillators/FM3Oscillator.h
+  dsp/oscillators/ModernOscillator.cpp
+  dsp/oscillators/ModernOscillator.h
+  dsp/oscillators/OscillatorBase.h
+  dsp/oscillators/OscillatorCommonFunctions.h
+  dsp/oscillators/SampleAndHoldOscillator.cpp
+  dsp/oscillators/SampleAndHoldOscillator.h
+  dsp/oscillators/SineOscillator.cpp
+  dsp/oscillators/SineOscillator.h
+  dsp/oscillators/StringOscillator.cpp
+  dsp/oscillators/StringOscillator.h
+  dsp/oscillators/TwistOscillator.cpp
+  dsp/oscillators/TwistOscillator.h
+  dsp/oscillators/WavetableOscillator.cpp
+  dsp/oscillators/WavetableOscillator.h
+  dsp/oscillators/WindowOscillator.cpp
+  dsp/oscillators/WindowOscillator.h
+  dsp/utilities/DSPUtils.cpp
+  dsp/utilities/DSPUtils.h
+  dsp/utilities/FastMath.h
+  dsp/utilities/LanczosResampler.cpp
+  dsp/utilities/LanczosResampler.h
+  dsp/utilities/SSEComplex.h
+  dsp/utilities/SSESincDelayLine.h
+  dsp/vembertech/basic_dsp.cpp
+  dsp/vembertech/basic_dsp.h
+  dsp/vembertech/halfratefilter.cpp
+  dsp/vembertech/halfratefilter.h
+  dsp/vembertech/lipol.cpp
+  dsp/vembertech/lipol.h
+  dsp/vembertech/portable_intrinsics.h
+  dsp/vembertech/shared.h
+  dsp/vembertech/vt_dsp.h
+  dsp/vembertech/vt_dsp_endian.h
+  globals.h
+  resource.h
+  version.cpp.in
+  version.h
+)
+
+if(APPLE)
+  target_compile_definitions(${PROJECT_NAME} PUBLIC
+    MAC=1
+    MAC_COCOA=1
+    COCOA=1
+    OBJC_OLD_DISPATCH_PROTOTYPES=1
```

Comment:
I'm 99% sure we only need MAC=1 now and the rest are from vstgui days

Reply:
Makes sense. Always glad to get rid of cruft!